converter/src/main/java/gov/cms/qpp/conversion/correlation/model/Config.java

```diff
@@ -0,0 +1,55 @@
+package gov.cms.qpp.conversion.correlation.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+	"decodeLabel",
+	"encodeLabels",
+	"goods"
+})
+public class Config {
+
+	@JsonProperty("decodeLabel")
+	private String decodeLabel;
+	@JsonProperty("encodeLabels")
+	private List<String> encodeLabels = new ArrayList<String>();
```

Comment:
We don't really need to specify `String` again in `new ArrayList<String>()`.  It can just be `<>`.  This applies to the other classes in this `model` package too.

Reply:
Yeah, this was autogenerated code, didn't think to go through and clean it up. Good catch.